Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/timeline.js

```diff
@@ -583,65 +677,65 @@ class TimelineFromEndpoint {
         DOM.inject(element, 
             `<div class="row">
                 ${this.bugTrackers.map(bugTracker => {
-                    const buttonText = `${bugTracker[0].toUpperCase()}${bugTracker.substring(1)}`;
-                    const buttonRef = REF.createRef({
-                        state: {
-                            loading: false
-                        },
-                        onStateUpdate: (element, stateDiff) => {
-                            if (stateDiff.loading)
-                                element.innerText = 'Waiting...';
-                            else
-                                element.innerText = buttonText;
-                        }
-                    });
-        
-                    buttonRef.fromEvent('click').action(e => {
-                        const requestPayload = {
-                            selectedRows: [],
-                            willFilterExpected: InvestigateDrawer.willFilterExpected,
-                            repositories: this.repositories,
-                            suite: this.suite,
-                            test: this.test
-                        };
-                        Array.from(this.selectedDots.keys()).forEach(config => {
-                            const dots = this.selectedDots.get(config);
-                            requestPayload.selectedRows.push({
-                                config,
-                                results: dots
-                            });
+                const buttonText = `${bugTracker[0].toUpperCase()}${bugTracker.substring(1)}`;
```

Comment:
nit: this is formatting churn. You should indent this back to its original indentation level.